v2.1/show-databases.md

```diff
@@ -26,15 +25,18 @@ No [privileges](privileges.html) are required to list the databases in the Cockr
 ~~~
 
 ~~~
-+--------------------+
-|      Database      |
-+--------------------+
-| bank               |
-| system             |
-+--------------------+
-(5 rows)
++---------------+
+| database_name |
++---------------+
+| defaultdb     |
+| postgres      |
+| system        |
++---------------+
+(3 rows)
 ~~~
 
+{% include {{page.version.version}}/default-databases.md %}
```

Comment:
Instead of embedding this as a callout following the example, I'd recommend we add an additional `Default Databases` section above the example. It'd make it much easier to cross-reference, if that's a need at any point. Thoughts?